css-sass.md

```diff
@@ -154,13 +159,61 @@ For example:
     // And definitely don't do this
     .sidebar .widget, .site_footer .widget { border-bottom: 1px solid #333;padding-bottom: 1em;margin-bottom: 1em;}
 
-### Group like rules
-
-Group your CSS rules in similar chunks. For example, put everything to do with fonts together, as with box model, positioning, and so forth.
+### Rule order
+
+Style rules should be placed in the following order. Note that anything bringing in rules not declared presently are placed first so you know what you're bringing in.
+
+1.  includes
```

Comment:
at first I wasn't sure what this meant.  Could we write `@include`s or `mixins`?